python/triton/experimental/gluon/language/amd/_layouts.py

```diff
@@ -37,35 +36,37 @@ class AMDMFMALayout(DistributedLayout):
     - 4: gfx950
     """
     version: int
+    warps_per_cta: List[int]
```

Comment:
Can we keep the original order because of ergonomics 1) it's arranging fields from smaller scape to larger (intrinsic -> warp -> cta), 2) it's considering frequency of customization and omission, e.g., we are more likely to use `tiles_per_warp` than the ctas_per_cga stuff` so at where it's used we aren't forced to provide less freqency pieces, 3)  it's more consistent with wmma layout?

Need to align ordering in comments and other functions too.